posthog/api/test/test_trends.py

```diff
@@ -120,29 +120,81 @@ class TrendsRequest:
     date_to: str
     interval: str
     insight: str
-    breakdown: Union[List[int], str]
-    breakdown_type: str
     display: str
     events: List[Dict[str, Any]]
 
 
-def get_trends(client, request: TrendsRequest, team: Team):
-    return client.get(
-        f"/api/projects/{team.id}/insights/trend/",
-        data={
-            "date_from": request.date_from,
-            "date_to": request.date_to,
-            "interval": request.interval,
-            "insight": request.insight,
-            "breakdown": request.breakdown,
-            "breakdown_type": request.breakdown_type,
-            "display": request.display,
-            "events": json.dumps(request.events),
-        },
-    )
+@dataclasses.dataclass
+class TrendsRequestBreakdown(TrendsRequest):
+    breakdown: Union[List[int], str]
+    breakdown_type: str
+
+
+def get_trends(client, request: Union[TrendsRequestBreakdown, TrendsRequest], team: Team):
+    data: Dict[str, Any] = {
+        "date_from": request.date_from,
+        "date_to": request.date_to,
+        "interval": request.interval,
+        "insight": request.insight,
+        "display": request.display,
+        "events": json.dumps(request.events),
+    }
+
+    if isinstance(request, TrendsRequestBreakdown):
+        data["breakdown"] = request.breakdown
+        data["breakdown_type"] = request.breakdown_type
+
+    return client.get(f"/api/projects/{team.id}/insights/trend/", data=data,)
 
 
 def get_trends_ok(client: Client, request: TrendsRequest, team: Team):
     response = get_trends(client=client, request=request, team=team)
     assert response.status_code == 200, response.content
     return response.json()
+
+
+@dataclasses.dataclass
+class NormalizedTrendResult:
+    value: float
+    label: str
+    person_url: str
+    breakdown_value: Optional[Union[str, int]]
+
+
+def get_trends_time_series_ok(
```

Comment:
non-blocking

When it's just pulling out detail from a repsonse, we can write tests like

```
trends_response = get_trends_ok(...)
time_series = get_time_series_from_trends_response(trends_response)
```

Personally I think that reads a little more clearly about what's going on. Otherwise I was thinking there must be another endpoint for "time series trends"

That's super minor though